pyproject.toml

```
@@ -1,11 +1,94 @@
 [build-system]
 requires = [
-    "setuptools>=42",
-    "scikit-build~=0.18.0",
+    "scikit-build-core>=0.10.7",
     "Cython >=3.0.11, <3.1.0"
 ]
-build-backend = "backend"
-backend-path = ["_custom_build"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "RapidFuzz"
+dynamic = ["version"]
+requires-python = ">= 3.9"
+authors = [
+  {name = "Max Bachmann", email = "pypi@maxbachmann.de"},
+]
+description = "rapid fuzzy string matching"
+readme = "README.md"
+classifiers=[
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "License :: OSI Approved :: MIT License",
+]
+Homepage = "https://github.com/rapidfuzz/RapidFuzz"
+Documentation = "https://rapidfuzz.github.io/RapidFuzz/"
+Repository = "https://github.com/rapidfuzz/RapidFuzz.git"
+Issues = "https://github.com/rapidfuzz/RapidFuzz/issues"
+Changelog = "https://github.com/rapidfuzz/RapidFuzz/blob/main/CHANGELOG.rst"
+
+[project.optional-dependencies]
+all = [
+  "numpy"
+]
+
+[project.entry-points.pyinstaller40]
+hook-dirs = "rapidfuzz.__pyinstaller:get_hook_dirs"
+tests = "rapidfuzz.__pyinstaller:get_PyInstaller_tests"
+
+[tool.scikit-build]
+minimum-version = "build-system.requires"
+sdist.include = [
+  "src/rapidfuzz/*.cxx",
+  "src/rapidfuzz/distance/*.cxx",
+]
+sdist.exclude = [
+  ".github"
+]
+wheel.exclude = [
+  "**.pyx",
+  "**.cxx",
+  "**.pxd",
+  "**.cpp",
+  "**.hpp",
+  "**.h",
+  "CMakeLists.txt",
+  "generate.sh"
+]
+wheel.packages = ["src/rapidfuzz"]
+wheel.cmake = false
+messages.after-success = "{yellow}Cmake unavailable, falling back to pure Python Extension"
```

```suggestion
messages.after-success = "{yellow}CMake unavailable, falling back to pure Python Extension"
```
Unnecessary stylistic nitpick. ;)

---

Fixed :)